### ISSUE TITLE
Informação ao cidadão por SMS

### ISSUE THREAD

**Issue by raseng** [SERMIL/sermilweb#42]
Seria possível acrescentar o telefone celular  do cidadão a fim de viabilizar a resposta por SMS?
Existe custo para realizar a resposta por SMS como fazem as empresas de avião, como a Gol? 
Eles chegaram ao requinte de dar a possibilidade de escolher como vai receber as respostas.

**Comment by wagnerlopes** [comment]
Sim, podemos incluir essa funcionalidade. Vou deixar aberta esta solicitação assim podemos estudar e colocar no sistema a demanda.
